### PR TITLE
Smssdoc

### PR DESCRIPTION
**Pull request by vera-chan**
Update the READMEs to include the SMSS scenario. I think this will address part of Issue 169

### CODE REVIEW

**Comment by vera-chan** [comment]
@tom-foyle Hi Tom, as Stew is away, I wonder whether you could review this issue as you were part of the discussion before. This PR will partly address the issues in #169 but I couldn't find the source file of the generated doc (https://ibm.github.io/cics-bundle-maven/plugin-info.html). I wonder whether that needs to be updated in the code? Thanks.

File: README.md [hunk 1]
```diff
@@ -69,10 +69,9 @@ To use the plugin to build CICS bundles, make sure that Maven is installed.
...
 
-However, if you are using the `deploy` goal of the plugin to deploy bundles to CICS, you must enable the CICS bundle deployment API. The CICS bundle deployment API is supported by the CMCI JVM server that must be set up in a WUI region (consult the [CICS TS doc](https://www.ibm.com/docs/en/cics-ts/5.6?topic=succs-configuring-cmci-jvm-server-cics-bundle-deployment-api) for details). To use the `deploy` goal, make sure that:
- * You have a CICS region that is at CICS® TS V5.6 or later
- * This region is configured to be a WUI region for the CICSplex that contains the deployment target region
- * This WUI region is configured to use the CMCI JVM server, including the CICS bundle deployment API
+However, if you are using the `deploy` goal of the plugin to deploy bundles to CICS, you must enable the CICS bundle deployment API. The CICS bundle deployment API is supported by the CMCI JVM server that must be set up in a WUI region or a single CICS region to turn it into an SMSS. See the [CICS TS doc](https://www.ibm.com/docs/en/cics-ts/6.1_beta?topic=suc-configuring-cmci-jvm-server-cics-bundle-deployment-api) for details. To use the `deploy` goal, make sure that:
```
Comment (tom-foyle) [inline README.md:72]: I'm not sure that "to turn it into an SMSS" is necessary here. It's possible that the region is already an SMSS if it's using "traditional" CMCI but not CMCI JVM server.

**Comment by tom-foyle** [comment]
> 
> 
> @tom-foyle Hi Tom, as Stew is away, I wonder whether you could review this issue as you were involved in the discussion before. This PR will partly address the issues in #169 but I couldn't find the source file of the generated doc (https://ibm.github.io/cics-bundle-maven/plugin-info.html). I wonder whether that needs to be updated in the code? Thanks.

Looks like that doc page is auto generated. You can change the parameter descriptions by changing the comments in here:
https://github.com/IBM/cics-bundle-maven/blob/573f9e36b23d4c479d7927fe43218c9cb51e1151/cics-bundle-maven-plugin/src/main/java/com/ibm/cics/cbmp/BundleDeployMojo.java#L76

File: README.md (hunk 1 above)
Comment (vera-chan) [reply README.md:72]: Removed it

**Comment by vera-chan** [comment]
@tom-foyle Hi Tom, I just pushed another commit to this PR (d0fbac9083d3f9a466e7af21af1db7e5e39b0a6d) to address your comments:
1. remove "to turn it into an smss"
2. update the description in bundledeploymojo.java to specify <region> and <cicsplex> are required for CICSPlex SM environments.
Could you review? Thanks.

**Comment by vera-chan** [comment]
Force pushed again with signed-off commit

**Review by tom-foyle — APPROVED** [review]

**Pull request merged**